packages/eslint/README.md

```diff
@@ -46,7 +46,7 @@ neutrino.use(eslint, {
     cwd: neutrino.options.root,
     useEslintrc: false,
     root: true,
-    extensions: ['js', 'jsx'],
+    extensions: neutrino.options.extensions,
```

Comment:
This means eslint will now be run against `.json` files too. Is this intended? (For example if people have the "enforce trailing commas" eslint rule enabled, eslint will complain unless the .json is changed to be invalid)

Reply:
Have we tested that this really happens? That would be weird! It would also mean single quote properties and values would also break...I guess we should find out. If it does, we can filter `json` from the list. Good catch.